ledger/participant-integration-api/src/main/scala/platform/indexer/JdbcIndexer.scala

```diff
@@ -29,9 +29,6 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.util.control.NonFatal
 
 object JdbcIndexer {
-  // TODO append-only: currently StandaloneIndexerServer (among others) has a hard dependency on this concrete class.
-  // Clean this up, for example by extracting the public interface of this class into a trait so that it's easier
-  // to write applications that do not depend on a particular indexer implementation.
```

Comment:
Has this comment become irrelevant? (asking just to get some context)

Reply:
Yes, I had added this comment, but decided that it would be overengineering to act on it now. We can do this later when we need this abstraction.

Reply:
Thanks!